pkg/podvolumebackup/util.go

```diff
@@ -0,0 +1,20 @@
+package podvolumebackup
```

Comment:
In v1.10 branch and main, we've already had a folder called `podvolume` where all the pod volume backup/restore related code reside in. Therefore, it seems to be redundant to add another `podvolumebackup` and `podvolumerestore`.
Moreover, the two `UpdateStatusToFailed` functions work for podvolumebackup/podvolumerestore CR only, they don't look like common functions that could be used by others in future.

Therefore, I suggest we still put the two functions in the controller files.

Reply:
Thanks for feedback. Will address it my next morning.